File: task-03/js/subtask-2.js
const fs = require('fs');
const readline = require('readline');

function subtask2() {
  const inputFilePath = 'input.txt';
  const outputFilePath = 'output.txt';

  const content = fs.readFileSync(inputFilePath, 'utf8');
  fs.writeFileSync(outputFilePath, content);
  const final = fs.readFileSync(outputFilePath, 'utf8');
  return final;
}

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Content of the file: \n', (inputContent) => {
  fs.writeFileSync('input.txt', inputContent);
  console.log('printing output.txt...');
  const result = subtask2();
  console.log(result);
  rl.close();
});

File: task-03/cpp/subtask-2.cpp
#include <iostream>
#include <fstream>
#include <string>

void subtask2() {
    std::string content;
    std::ifstream infile("input.txt");
    std::getline(infile, content);
    infile.close();

    std::ofstream outfile("output.txt");
    outfile << content;
    outfile.close();

    std::ifstream readfile("output.txt");
    std::getline(readfile, content);
    readfile.close();

    std::cout << content << std::endl;
}

int main() {
    std::string inputcontent;
    std::cout << "Content of the file: " << std::endl;
    std::getline(std::cin, inputcontent);

    std::ofstream infile("input.txt");
    infile << inputcontent;
    infile.close();

    std::cout << "printing output.txt..." << std::endl;
    subtask2();

    return 0;
}
